uktrade/data-hub-frontend
# Mock postcode lookup call to reduce test flakiness

## Description of change

This PR mocks out the call the postcode lookup service to avoid intermittent failing tests as shown here - https://dashboard.cypress.io/projects/w97se2/runs/4606/specs

This means that in your sandbox environment you will need to set the following env var:

`DATA_STORE_SERVICE_POSTCODE=http://localhost:8001/sandbox/postcodelookup/`

I have added the necessary env var into the vault so it is no longer hard coded. 

## Test instructions

Tests should all pass and you shouldn't see the add-company.spec failing every now and then because of a slow response from `https://api.getAddress.io/`

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
